package.json

```diff
@@ -32,11 +36,15 @@
     "@ionic/storage-angular": "^3.0.6",
     "@ngx-translate/core": "^13.0.0",
     "@ngx-translate/http-loader": "^6.0.0",
+    "ajv": "^6.12.6",
```

Comment:
What is this plugin?

Reply:
while installing file-transfer plugin i was getting error message like ajv and ajv-keywords should be more than 6.*.* something so i added this